boards/ssrc/saluki/src/CMakeLists.txt

```diff
@@ -63,6 +64,7 @@ else()
 		toc.c
 		i2c.cpp
 		spi.cpp
+		mpfs_emmcsd.c
```

Comment:
Does not work for me. Need to add 
${NUTTX_DIR}/boards/risc-v/mpfs/icicle/src/mpfs_emmcsd.c